intellij-ide.md

```diff
@@ -188,8 +188,22 @@ types of code changes cannot be hot reloaded though:
 For these changes you can fully restart your application, without having to end
 your debugging session:
 
-1. In the Debug panel at the bottom, click the **Full Application Restart button**
-(shown with an icon containing a circular arrow).
+1. Don't click the Stop button; simply re-click the Run button (if in a run
+session) or Debug button (if in a debug session).
+
+## Editing Android code in IntelliJ IDEA {#edit-android-code}
+
+To enable editing of Android code in IntelliJ IDEA, you need to configure the
+location of the Android SDK:
+
+1. Right-click the **android** folder in the Project view, and select **Open
+Module Settings**.
+1. In **Sources** tab, locate the **Language level** field, and select level '8'
+or later.
+1. In **Dependencies** tab, locate the **Module SDK** field, and select an
+Android SDK. If no SDK is listed, click **New...** and specify the location of
+the Android SDK.
```

Comment:
"This should be the same Android SDK shown by 'flutter doctor' in the 'Android toolchain' section."

(Or something like that. Do we need to explain how to run flutter doctor?)